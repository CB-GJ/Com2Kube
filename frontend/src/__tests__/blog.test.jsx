import React from "react"
import renderer from "react-test-renderer"
import Blog from "../pages/blog"

describe("Blog", () => {
  test("renders correctly enzyme", () => {
    const wrapper = renderer.create(<Blog />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })

  test("return successful results", () => {
    fetchMock.get("https://dev.to/api/articles?username=com2kube", {
      title: "DevOps Handy Tool: Com2kube"
    })
    const response = fetch("https://dev.to/api/articles?username=com2kube")
    const result = response.json()
    expect(result.title).toEqual("DevOps Handy Tool: Com2kube")
  })
})
