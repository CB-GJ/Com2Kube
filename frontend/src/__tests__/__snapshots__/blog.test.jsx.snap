// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly enzyme 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-body-3"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
    />
  </div>
</div>
`;
